### PR TITLE
OKTA-731717 - Update number of inline hooks

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updating number of inline hooks available per org from 50 to 100
- **Is this PR related to a Monolith release?** `2024.06.0`

### Resolves:

* [OKTA-731717](https://oktainc.atlassian.net/browse/OKTA-731717)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/230e2675be0ed3f81029c747fe84592776e9a471/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/29b88cec-8b28-4e10-999f-e582b4690767) | 81  | 56  | 72  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md](https://github.com/okta/okta-developer-docs/blob/230e2675be0ed3f81029c747fe84592776e9a471/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fhooks-best-practices%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6c7732aa-2cc6-4cb0-8939-81d96fa01c95) | 85  | 57  | 85  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ea513f6c-688b-4c29-9e77-7268a3e1a388?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Your external service that processes hook requests must consider that the order
...
 |            | Maximum number of event hooks per org | 10 | A maximum of 10 active event hooks can be configured per org. You can configure each event hook to deliver multiple event types. |
 | Inline hook | Timeout | 3 seconds | Inline hooks have a completion timeout of three seconds with a single retry. However, a request isn't retried if your endpoint returns a 4xx HTTP error code. Any 2xx code is considered successful, and the request is not retried. If the external service endpoint responds with a redirect, it isn't followed. |
-|             | Maximum number of inline hooks per org | 50 | The maximum number of inline hooks that can be configured per org is 50, which is a combined total for any combination of inline hook types. |
+|             | Maximum number of inline hooks per org | 100 | The maximum number of inline hooks that can be configured per org is 100, which is a combined total for any combination of inline hook types. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md:70]:
```suggestion
|             | Maximum number of inline hooks per org | 100 | The maximum number of inline hooks that you can configure per org is 100, which is a combined total for any combination of inline hook types. |
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/a0b3d0c3bb74547d1aa9995e1b0bc5a81f6a822a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/903f1e73-8a36-426a-9677-6fc37d7e1638) | 81  | 56  | 72  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md](https://github.com/okta/okta-developer-docs/blob/a0b3d0c3bb74547d1aa9995e1b0bc5a81f6a822a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fhooks-best-practices%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2e831273-c8b1-4dc0-90ca-4acb9d439eb7) | 85  | 57  | 85  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b62d57a5-014e-45a0-9a2c-2aba9aeb8712?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**